pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,32 +132,71 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterStorageClassChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	oldSC, newSC, err := util.As[*storagev1.StorageClass](oldObj, newObj)
+	if err != nil {
+		return framework.Queue, err
+	}
+
```

Comment:
It's forbidden to update the volume binding mode of a storage class. If a storage class's binding mode isn't VolumeBindingWaitForFirstConsumer, when it is created or updated, the action itself doesn't make the pod become schedulable directly. If then some PVCs is bound by PVs due to the storage class (re-)created, it should be reflected in the PVC update event. so in this function, we should only take care of special storage classes with the VolumeBindingWaitForFirstConsumer binding mode.

Reply:
If we only consider VolumeBindingWaitForFirstConsumer, this PR will depend on the other hint func which handle PVC events. 

So let's merge another firstly once that PR is ready to be merged. @sanposhiho

Reply:
Thank you for your suggestion. In that case, would you mind reviewing the following PR first, which adds QHint for PVC?
https://github.com/kubernetes/kubernetes/pull/124959